docs/components/receive_route.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Receive-route
+type: docs
+menu: components
+---
+
+# Receive-route
+
+The `thanos receive-route` command implements the [Prometheus Remote Write API](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write). It receives metrics from the target and forwards them to the right [receiver](./receive.md) in the hashring. It also buffers the incoming requests and handles changes in hashring configurations.
+We recommend this component to users who can only push into a Thanos due to air-gapped, or egress only environments. Please note the [various pros and cons of pushing metrics](https://docs.google.com/document/d/1H47v7WfyKkSLMrR8_iku6u9VB73WrVzBHb2SB6dL9_g/edit#heading=h.2v27snv0lsur).
+
+For further information on tuning Prometheus Remote Write [see remote write tuning document](https://prometheus.io/docs/practices/remote_write/).
+
+# Example
+
+```bash
+thanos receive-route \
+    --http-address 0.0.0.0:10909 \
+    --receive.replication-factor 1 \
+    --receive.hashrings-file ./data/hashring.json \
+    --remote-write.address 0.0.0.0:10908 \
+```
+
+The example of `remote_write` Prometheus configuration:
+
+```yaml
+remote_write:
+- url: http://<thanos-receive-container-ip>:10908/api/v1/receive
```

Comment:
We should point Prometheus at the router rather than directly at the receiver:
```suggestion
- url: http://<thanos-receive-route-container-ip>:10908/api/v1/receive
```